### DISCUSSION TITLE
How to properly cite a figure in Quarto?

### DISCUSSION THREAD

**Discussion by MohammedAliSharafuddin** [quarto-dev/quarto-cli#7250] [Q&A]
### Description

Apologies in advance if these are silly questions. I am new to Quarto and using it to compile a teaching material. I need some guidance with,

(1) Including citation for figures.
For example: If we use 
```{latex}
![MSC GÜLSÜN [@tornEersteBezoekAan2019]](images/MSC.jpg){#fig-MSC-GÜLSÜN}, 
```
the citation also appears in the list of figures at the book index. How to avoid it. I need only the name of the figure and not the citation in the "List of Figures"
![image](https://github.com/quarto-dev/quarto-cli/assets/50414472/f3ba5f0f-85ae-487f-b574-242db7c26aff)

(2) The second issue is the same as  "https://stackoverflow.com/questions/72799825/side-by-side-caption-positions-of-figure-and-table". Using the codes chunk makes an additional caption. Looking for a solution to include a plot and table in a single chunk. 

```{r}
#| layout-ncol: 2
#| layout-valign: bottom
#| fig-cap: "A plot"
#| tbl-cap: "A table"

library(knitr)

# plot on the left
plot(cars)

# table on the right
kable(head(cars))
```

Thanks

**Comment by mcanouil** [comment]
1. I am not sure there is a way. In pure LaTeX, this can be done via `\caption*` but not sure how to do this in Quarto.

2. What exactly do you want? Having the figure and the table side by side? Have them as sub element of something you can cross-reference?
Why do you want them to be in the same code cell?

The following works for HTML but not for LaTeX `LaTeX Error: Not in outer par mode.`, it seems the latex document is malformed:

````qmd
---
format:
  pdf: default
  html: default
execute:
  echo: false
---

::: {layout-ncol="2" layout-valign="bottom"}

```{r}
#| label: fig-example
#| fig-cap: Figure caption
plot(cars)
```

```{r}
#| label: tbl-example
#| tbl-cap: Table caption
knitr::kable(head(cars))
```

:::
````

**Reply by mcanouil** [reply]
The issue is that this adds a figure/table environment inside a minipage which is not allowed in LaTeX.
See:
- #7262

**Reply by MohammedAliSharafuddin** [reply]
Let me reframe the question.

1. How to properly cite a figure in Quarto? I am not meaning the figure caption and cross-referencing within the document which is already well explained in Quarto documents. I am asking about the alternative for \cite command in Quarto for figures.

Lets say, in Latex, we can straighaway insert the \cite command in a figure environment. 
```{=latex}
\begin{figure}[!htbp]
	\centering
	\includegraphics[width=0.75\linewidth]{CIF.png}
	\caption{CIF (Cost, Insurance and Freight)}
	Source: \cite{noauthor_incoterms_nodate}
	\label{fig:CIF (Cost, Insurance and Freight)}
\end{figure}
```
From this discussion https://stackoverflow.com/questions/47860390/how-to-cite-within-a-figure-caption-w-rmarkdown-native-citation it seems to be resolved with rmarkdown and working.

So, the same also works in Quarto when we insert a citation from bib file using [@thereference].  
```{=latex}
![Elephant [@elephant2022]](elephant.png){fig-align="left"}
```
But the issue is with list of figures in the index of the PDF book. The [@elephant2022] information also appears along with the figure caption in the list of figures which is not a standard format. So, how to avoid [@elephant2022] when generating list of figures in a Quarto (PDF) document.

![image](https://github.com/quarto-dev/quarto-cli/assets/50414472/6af7f805-daab-473d-9aa0-0b9f39205189)


2. Never mind about the figure and table in a quarto envionment. I will use a Latex environment for now.

**Reply by mcanouil** [reply]
Please use code blocks and inline code using classic markdown syntax and using the menu bar in the editor.
It will make your post more legible while also avoiding tagging people/organisation for no reasons.

Note that you can use raw LaTeX using raw code blocks:  `` ```{=latex} ``
